fidesctl/fidesctl.toml

```diff
@@ -3,3 +3,4 @@ server_url = "http://fidesctl:8080"
 
 [api]
 database_url = "postgresql+psycopg2://postgres:fidesctl@fidesctl-db:5432/fidesctl"
+test_database_url = "postgresql+psycopg2://postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
```

Comment:
this is is a nit and won't block merge, but I think we should configure our IDEs to leave newlines at the end of files

Reply:
sure thing, just enabled it in my IDE